lib/atomlist.c

```diff
@@ -0,0 +1,348 @@
+/*
+ * Copyright (c) 2016-2018  David Lamparter, for NetDEF, Inc.
+ *
```

Comment:
I think it would be good to clearly delineate  who the copyright holder is here e.g., is it 
NetDef (with Author being David)
   or
David (with an acknowledgement to netdef for supporting the work)